src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-form/configuration-form.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="col-sm-12 col-lg-6">
+  <form name="configForm"
+        class="form-horizontal"
+        #formDir="ngForm"
+        [formGroup]="configForm"
+        novalidate>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <span><span i18n>Edit</span> {{ configForm.getValue('name') }}</span>
```

Comment:
I think you could use ``<ng-container i18n>Edit</ng-container>`` here instead of using ``<span>``.

Reply:
Do you mean 
```
<span><ng-container i18>Edit</ng-container> {{ configForm.getValue('name') }}</span>
```
so I need to keep the outer span?

Reply:
You can drop the outer span.

Reply:
Updated